src/group-configurations/experiment-configurations-section/ExperimentCard.test.jsx

```diff
@@ -0,0 +1,164 @@
+  });
+
+  it('expands/collapses the container experiment configuration on title click', () => {
+    const { queryByTestId, getByTestId } = renderComponent();
+    const cardTitle = getByTestId('configuration-card-header__button');
+    userEvent.click(cardTitle);
+    expect(queryByTestId('configuration-card-content')).toBeInTheDocument();
+
+    userEvent.click(cardTitle);
+    expect(queryByTestId('configuration-card-content')).not.toBeInTheDocument();
+  });
+
+  it('renders experiment configuration badge with used only one location', () => {
+    const { getByText } = renderComponent();
+    expect(
+      getByText(rootMessages.usedInLocation.defaultMessage),
+    ).toBeInTheDocument();
+  });
+
+  it('renders experiment configuration badge with used locations', () => {
+    const fewLocationsArray = [
+      {
+        label: 'Unit1name / Content Experiment',
+        url: '/container/block-v1:2u+1+1+type@split_test+block@ccfae830ec9b406c835f8ce4520ae395',
+      },
+      {
+        label: 'UnitName 2 / Content Experiment',
+        url: '/container/block-v1:2u+1+1+type@split_test+block@ccfae830ec9b406c835f8ce4520ae396',
+      },
+    ];
+    const experimentConfigurationUpdated = {
+      ...experimentConfiguration,
+      usage: fewLocationsArray,
+    };
+    const { getByText } = renderComponent({
+      configuration: experimentConfigurationUpdated,
+    });
+    expect(
+      getByText(
+        rootMessages.usedInLocations.defaultMessage.replace(
+          '{len}',
+          experimentConfigurationUpdated.usage.length,
+        ),
+      ),
+    ).toBeInTheDocument();
+  });
+
+  it('renders experiment configuration without access to units', () => {
+    const experimentConfigurationUpdated = {
+      ...experimentConfiguration,
+      usage: [],
+    };
+    const { queryByText, getByTestId } = renderComponent({
+      configuration: experimentConfigurationUpdated,
+    });
+    expect(
+      queryByText(commonMessages.accessTo.defaultMessage),
+    ).not.toBeInTheDocument();
+
+    const cardTitle = getByTestId('configuration-card-header__button');
+    userEvent.click(cardTitle);
+    expect(
+      getByTestId('experiment-configuration-card-usage-empty'),
+    ).toBeInTheDocument();
+  });
+
+  it('renders usage with validation error message', () => {
+    const experimentConfigurationUpdated = {
+      ...experimentConfiguration,
+      usage: [{
+        label: 'Unit1name / Content Experiment',
+        url: '/container/block-v1:2u+1+1+type@split_test+block@ccfae830ec9b406c835f8ce4520ae395',
+        validation: {
+          type: 'warning',
+          text: 'This content experiment has issues that affect content visibility.',
+        },
+      }],
+    };
+    const { getByText, getByTestId } = renderComponent({
+      configuration: experimentConfigurationUpdated,
+    });
+
+    const cardTitle = getByTestId('configuration-card-header__button');
```

Comment:
[nit] I think this double underscore here is not needed. Let's refactor a bit.
```suggestion
    const cardTitle = getByTestId('configuration-card-header-button');
```

Reply:
changed